salt/metalk8s/macro.sls

```diff
@@ -3,11 +3,13 @@
 {%- macro pkg_installed(name='') -%}
   {%- set package = repo.packages[name] | default({}) %}
   pkg.installed:
-    - name: {{ name }}
-    - fromrepo: {{ repo.repositories.keys() | join(',') }}
-    {%- if package.version | default(None) %}
+    - name: {{ package.name | default(name) }}
+  {%- if package.version | default(None) %}
     - version: {{ package.version }}
     - hold: True
     - update_holds: True
-    {%- endif %}
+  {%- endif %}
+  {%- if grains['os'] == 'CentOS' %}
```

Comment:
What about RHEL? You'll want to dispatch on family, I guess.

Reply:
change done.